apps/api/src/main.ts

```diff
@@ -2,19 +2,24 @@ import { Logger } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app/app.module';
+import { ArangoConnectionProvider } from './persistence/database/arango-connection.provider';
 
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule, { cors: true });
-  const globalPrefix = 'api';
-  app.setGlobalPrefix(globalPrefix);
+    const app = await NestFactory.create(AppModule, { cors: true });
+    const globalPrefix = 'api';
+    app.setGlobalPrefix(globalPrefix);
 
-  // app.enableCors({});
+    // app.enableCors({});
```

Comment:
Delete this?  Or is there still a potential to go with this syntax over the options object in `.create()`